Assembly-CSharp/ModLoader.cs

```diff
@@ -235,8 +235,9 @@ Assembly Resolve(object sender, ResolveEventArgs args)
             UObject.DontDestroyOnLoad(version);
             
             UpdateModText();
-            
-            Logger.APILogger.LogDebug("Updated mod text.");
+
+            // Adding version nums to the modlog by default to make debugging significantly easier
+            Logger.APILogger.Log("Set mod text:\n" + modVersionDraw.drawString);
```

Comment:
Use `LogDebug`. If someone is having errors they'll want to set the log level to debug anyways. If they aren't, then this will just be a bunch of clutter as-is.

Reply:
The idea is that it's more for the casual user getting hard-to-reproduce errors and giving an error report to the dev, rather than the dev fixing a bug - but it's true that it adds clutter so I'm happy to change it if you prefer.

Reply:
That's fair but I think it'd be strange to have version text be logged when loading the mods from the assembly is still debug only. I could go for it if you want though. If you're doing that I'd change it to be finished loading mod instead of setting text because that seems to be the point of the logging